### ISSUE TITLE
a way to personalize the promise ring

### ISSUE THREAD

**Issue by dragonborn2252** [Monika-After-Story/MonikaModDev#3807]
how about giving the player the ability to add their birth stone to moni's promise ring she could think of it as a way to have little part of you with her at all times (I could not think of a better way to word that so sorry if it's confusing) so for example mine would be something like a topaz,Turquoise,Zircon or Tanzanite

**Comment by kaido1224** [comment]
Feasible once the below is done, depending on if someone makes the art for it. Until then, I don't see it happening.

https://github.com/Monika-After-Story/MonikaModDev/issues/3416

**Comment by ThePotatoGuy** [comment]
marking as duplicate of linked issue, also of the various personalized ring issues.
